documentation/homework/HomeWork3/todo-app/src/components/DeletedTask.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const DeletedTask = props => {
+    return ( <li className = "task mdc-elevation--z1"
+        key = {
+            props.id
+        } >
+        <p className = {
+            props.completed ? "completed" : null
+        } > {
+            props.name
+        } </p> <div className = "icon-group" >
+        <i className = "material-icons"
+        onClick = {
+            props.undoDelete.bind(null, props.id)
+        } > undo </i> <i className = "material-icons"
+        onClick = {
+            props.deleteForever.bind(null, props.id)
+        } > delete_forever </i> </div> </li>
+    );
+};
```

Comment:
hmm not sure what are you trying to do here! is it a confirmation message? to cancel or delete?
if so, you dont have to do anything (like filter out the array) before making the final decision!
correct me if i'm wrong!